airbyte-integrations/connectors/source-trustpilot/source_trustpilot/source.py

```diff
@@ -2,88 +2,17 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 #
 
+from airbyte_cdk.sources.declarative.yaml_declarative_source import YamlDeclarativeSource
 
-from typing import Any, List, Mapping, MutableMapping, Optional, Tuple
+"""
+This file provides the necessary constructs to interpret a provided declarative YAML configuration file into
+source connector.
 
-import pendulum
-from airbyte_cdk.models import SyncMode
-from airbyte_cdk.sources import AbstractSource
-from airbyte_cdk.sources.streams import Stream
+WARNING: Do not modify this file.
+"""
 
-from .auth import TrustpilotApikeyAuthenticator, TrustpilotOauth2Authenticator
-from .streams import BusinessUnits, ConfiguredBusinessUnits, PrivateReviews
 
-
-class SourceTrustpilot(AbstractSource):
-    def __init__(self, *args, **kargs):
-        super().__init__(*args, **kargs)
-        self.__public_auth_params: Mapping[str, Any] = {}
-        self.__oauth2_auth_params: Mapping[str, Any] = {}
-        self.__configured_business_units_stream: Optional[ConfiguredBusinessUnits] = None
-
-    def _public_auth_params(self, config: MutableMapping[str, Any]):
-        """
-        Creates the authorization parameters for the Trustpilot Public API.
-
-        The Public API only requires the API key (stored on the credentials/client_id).
-        It does not require OAuth2 authentication.
-
-        See also: https://documentation-apidocumentation.trustpilot.com/#Auth
-        """
-        if not self.__public_auth_params:
-            auth = TrustpilotApikeyAuthenticator(token=config["credentials"]["client_id"])
-            self.__public_auth_params = {"authenticator": auth}
-        return self.__public_auth_params
-
-    def _oauth2_auth_params(self, config: MutableMapping[str, Any]):
-        if not self.__oauth2_auth_params:
-            auth = TrustpilotOauth2Authenticator(
-                config, token_refresh_endpoint="https://api.trustpilot.com/v1/oauth/oauth-business-users-for-applications/refresh"
```

Comment:
@btkcodedev I think this is where the original refresh URL was — the refresh URL in the low-code manifest is different, so that could just be the bug here.